### PR TITLE
test: add new test cases for --encoding option of the export command

### PR DESCRIPTION
**Pull request by hung-cybo**
## Why

Adding export e2e test cases.

## What

- adding test cases to verify `--encoding` option
  - `--encoding` utf8
  - `--encoding` sjis
  - unsupported character code

## How to test

```
pnpm build
pnpm test:e2e
```

## Checklist

- [x] Read [CONTRIBUTING.md](https://github.com/kintone/cli-kintone/blob/main/CONTRIBUTING.md)
- [ ] Updated documentation if it is required.
- [x] Added/updated tests if it is required. (or tested manually)
- [x] Passed `pnpm lint` and `pnpm test` on the root directory.

### CODE REVIEW

File: features/export.feature [hunk 1]
```diff
@@ -412,3 +412,36 @@ Feature: cli-kintone export command
     When I run the command with args "record export --base-url $$TEST_KINTONE_BASE_URL --app $APP_ID --api-token $API_TOKEN --guest-space-id 9999999999999999999"
     Then I should get the exit code is non-zero
     And The output error message should match with the pattern: "\[403] \[CB_NO02] No privilege to proceed"
+
+  Scenario: CliKintoneTest-111 Should return the record contents when exporting the record with --encoding option is utf8.
```
Comment (tuanphamcybozu) [inline features/export.feature:415]:
```suggestion
  Scenario: CliKintoneTest-111 Should return the record contents when exporting the record with --encoding option as utf8.
```

File: features/step_definitions/export.ts [hunk 2]
```diff
@@ -79,7 +80,21 @@ Then(
...
+
+Then(
+  "The output message with {string} encoded should match with the data below:",
+  async function (encoding, table) {
+    const decodedOutputMsg = iconv.decode(this.response.stdout, encoding);
+    const records = this.replacePlaceholdersInRawDataTables(table.raw());
+    records.forEach((record: string[]) => {
+      const values = record
+        .map((field: string) => (field ? `"${field}"` : ""))
+        .join(",");
```
Comment (tasshi-me) [inline features/step_definitions/export.ts:96]: Can we make these lines reusable?
It is the same as the step "The output message should match the data in the order as below:"

**Review by tasshi-me — APPROVED** [review]
LGTM!

File: features/step_definitions/export.ts (hunk 2 above)
Comment (hung-cybo) [reply features/step_definitions/export.ts:96]: @mshrtsr 

Currently, some other places are also using the above logic.
I created a new PBI to refactor all of them.

**Pull request merged**